lib/Doctrine/DBAL/Schema/Comparator.php

```diff
@@ -488,8 +488,10 @@ public function diffColumn(Column $column1, Column $column2)
         $platformOptions1 = $column1->getPlatformOptions();
         $platformOptions2 = $column2->getPlatformOptions();
 
-        foreach (array_keys(array_intersect_key($platformOptions1, $platformOptions2)) as $key) {
-            if ($properties1[$key] !== $properties2[$key]) {
+        foreach (array_merge(array_keys($platformOptions1), array_keys($platformOptions2)) as $key) {
```

Comment:
Hmmm this does not seem to be working (see Travis). IMO the logic should be as follows: Do an `array_diffkey()` and mark all key differences as changed, do an `array_intersect_key()` and mark all keys as changed where the values do not match. However I am not sure how this will behave with "default values" like collation which DBAL sets implicitly in the platform (MySQL's default collation for tables for example) if not explicitly set. This could lead to undesired diffs, too. So you might have to play around a bit and get a working test environment going on your machine.